projects/plugins/social/src/js/utils/api.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { select } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { STORE_ID } from '../store';
+
+export const fetchEndpoint = ( { endpoint = '', method = 'GET', data = null } = {} ) => {
+	const store = select( STORE_ID );
+	const apiRootUrl = store.getAPIRootUrl();
```

Comment:
If we're using `apiFetch` it should be configured with the root URL, so we can pass `endpoint` as the `path`. In which case, this wrapper might not be needed at all.

Reply:
Ah neat, that seems to work! I'll get rid of the wrapper.

Reply:
Changed in [7052def](https://github.com/Automattic/jetpack/pull/24203/commits/7052deff193e6e7b12ddf4444d5ef122015d4ca8)